tests/FakeItEasy.Tests.Approval/ApiApproval.cs

```diff
@@ -39,23 +38,10 @@ public void ApproveApi(string projectName, string frameworkVersion)
             var assembly = Assembly.LoadFile(Path.GetFullPath(assemblyFile));
             var publicApi = ApiGenerator.GeneratePublicApi(assembly, options: null);
 
-            Approvals.Verify(
-                WriterFactory.CreateTextWriter(publicApi),
-                new ApprovalNamer(projectName, frameworkVersion),
-                Approvals.GetReporter());
-        }
-
-        private class ApprovalNamer : IApprovalNamer
-        {
-            public ApprovalNamer(string projectName, string frameworkVersion)
-            {
-                this.Name = frameworkVersion;
-                this.SourcePath = Path.Combine(GetSourceDirectory(), "ApprovedApi", projectName);
-            }
-
-            public string SourcePath { get; }
-
-            public string Name { get; }
+            var settings = new VerifySettings();
+            settings.UseDirectory("ApprovedApi/" + projectName);
+            settings.UseFileName(frameworkVersion);
+            return Verifier.Verify(publicApi, settings);

```

Comment:
Maybe use `await` instead of returning the task? Otherwise, if there's an exception thrown asynchronously, `ApproveApi` won't even be visible in the stack trace.

Reply:
I literally have no idea! I just copied stuff! Thanks!